swig/csharp/CMakeLists.txt

```diff
@@ -1,4 +1,10 @@
-include(GdalStandardIncludes)
+option(GDAL_CSHARP_ONLY "Compile C# bindings on existing GDAL installation" OFF)
+
+if (GDAL_CSHARP_ONLY)
+  find_package(GDAL REQUIRED)
+else (GDAL_CSHARP_ONLY)
```

Comment:
else (foo) / endif (foo) are deprecated practices in modern CMake. just use else() / endif(). Other occurences below

Reply:
Sorry - old habit. I will change in the next commit